llvm/lib/Target/AMDGPU/Utils/AMDGPUBaseInfo.cpp

```diff
@@ -607,6 +615,16 @@ bool isTrue16Inst(unsigned Opc) {
   return Info ? Info->IsTrue16 : false;
 }
 
+bool isInvalidSingleUseConsumerInst(unsigned Opc) {
+  const SingleUseExceptionInfo *Info = getSingleUseExceptionHelper(Opc);
+  return Info ? Info->IsInvalidSingleUseConsumer : false;
```

Comment:
Nit: `return Info && Info->IsInvalidSingleUseConsumer`. (I realize you were just copying the existing code.)

Reply:
Done